opencti-platform/opencti-front/tests_e2e/artifact/createArtifact.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { ArtifactPage} from "../model/Artifact.pageModel";
+import { ArtifactImportPage} from "../model/ArtifactImport.pageModel";
+import { expect, test } from "../fixtures/baseFixtures";
+
+
+test('Validate a new artifact file', async ({ page }) => {
```

Comment:
Be more explicit on what the test is testing: should display an error if trying to create an artifact without a file